src/test/groovy/io/seqera/wave/util/RegHelperTest.groovy

```diff
@@ -284,4 +283,23 @@ class RegHelperTest extends Specification {
                 .hash()
                 .toString()
     }
+
+    @Unroll
+    def 'should validate the provided custom container image name'(){
+        expect:
+        RegHelper.isValidImageName( IMAGENAME ) == ISVALID
+
+        where:
+        IMAGENAME               | ISVALID
+        'foo'                   | true
+        'foo:'                  | false
+        'foo:SHA123asdf'        | false
```

Comment:
I would edit these 2 to explicitly check about sha256 reference syntax:
```
'foo:SHA123asdf' -> 'foo@sha256:asdf'

'foo/bar/foo:SHA123asdf' -> 'foo/bar/foo@sha256:asdf'
cowsay@sha256:d24d
```

Reply:
done